src/components/ContentWidgets/ImagesInRow/ImagesInRow.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { ContentSingularData } from "../../_data/ContentSingularData";
+import "./ImagesInRow.css";
+
+/**
+ * Displays a row of images and corresponding captions
+ *
+ * Last Modified
+ * Nitesh
+ * July 13, 2019
+ */
+export const ImagesInRow: React.FC<ContentSingularData> = ({
+    imagesInRow_linkCaptionPairs
+}) => {
+    if (!imagesInRow_linkCaptionPairs) {
+        return <></>;
+    }
+    interface linkCaptionPair {
```

Comment:
Declare this as a type instead, and have it be outside the function declaration, not inside.